net/net.go

```diff
@@ -271,15 +271,13 @@ func (n *net) AddThread(ctx context.Context, addr ma.Multiaddr, opts ...core.New
 	}); err != nil {
 		return
 	}
-	if args.ThreadKey.CanRead() {
-		var linfo thread.LogInfo
-		linfo, err = createLog(n.host.ID(), args.LogKey)
-		if err != nil {
-			return
-		}
-		if err = n.store.AddLog(id, linfo); err != nil {
-			return
-		}
+	var linfo thread.LogInfo
```

Comment:
We want to avoid creating a log if it won't be used _ever_, since it will get pushed to the other peers.